Invoke-Locksmith.ps1

```diff
@@ -767,6 +789,52 @@ function Format-Result {
     }
 }
 
+function ConvertTo-IdentityReference {
```

Comment:
Need to add checks for Orphaned and Non-Windows-created SIDs like we do in other locations, but this is a good start!